docs/create-packages/Explicit-assembly-reference.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Explicit Assembly References
+description: Make a subset of assemblies in the package available to the compiler, while all assemblies are available at runtime.
+author: zivkan
+ms.author: zivkan
+ms.date: 05/24/2019
+ms.topic: conceptual
+---
+
+# Explicit Assembly References
+
+Explicit assembly references allows a subset of assemblies to be used for intellisense and compiling, while all assemblies are available at run-time. `PackageReference` and `packages.config` work differently, and as a result package authors need to take care to create the package to be compatible with both project types.
+
+> [!Note]
+> Explicit assembly references are related to .NET assemblies. It is not a method to distribute native assemblies that are P/Invoked by a managed assembly.
+
+## `PackageReference` support
+
+When a project uses a package with `PackageReference` and the package contains a `ref\<tfm>\` directory, NuGet will advertise those assembles as compile-time assets, while the `lib\<tfm>\` assemblies are advertised as runtime assets. Assemblies in `ref\<tfm>\` are not used at runtime. This means it is necessary for any assembly in `ref\<tfm>\` to have a matching assembly in either `lib\<tfm>\` or a relevant `runtime\` directory, otherwise runtime errors will likely occur. Since assemblies in `ref\<tfm>\` are not used at runtime, they may be [metadata-only assemblies](https://github.com/dotnet/roslyn/blob/master/docs/features/refout.md) to reduce package size.
```

Comment:
Is it possible to replace "advertise" in these several paragraphs with another word? I'd suggest "designate", "show", "classify", or "consider".